gqlalchemy/memgraph.py

```diff
@@ -236,11 +246,20 @@ def save_node(self, node: Node):
                 )
             elif len(matching_nodes) == 1:
                 node._id = matching_nodes[0]["node"]._id
-                return self.save_node_with_id(node)
+                result = self.save_node_with_id(node)
             else:
-                return self.create_node(node)
+                result = self.create_node(node)
         else:
-            return self.create_node(node)
+            result = self.create_node(node)
+
+        if self.on_disk_db:
+            for field in node.__fields__:
+                value = getattr(node, field, None)
+                if value is not None:
+                    if "on_disk" in node.__fields__[field].field_info.extra:
+                        self.on_disk_db.save_node_property(result._id, field, value)
+                        setattr(result, field, value)
```

Comment:
Maybe split this into new method => `_save_property_in_disk_db`

Reply:
yep, will do.